firmware/Core/Src/telecommands/telecommand_definitions.c

```diff
@@ -332,6 +333,17 @@ const TCMD_TelecommandDefinition_t TCMD_telecommand_definitions[] = {
         .readiness_level = TCMD_READINESS_LEVEL_FOR_OPERATION,
     },
 
+    // ****************** SECTION: agenda_telecommand_defs ******************
```

Comment:
Doesn't this section already exist in this file?

Reply:
Yeah it does, I fixed it and renamed it